adr/00-packages.md

```diff
@@ -0,0 +1,103 @@
+# Architecture Decision Record: Package structure
+
+## Context
+
+We need to specify structure for the application. 
+It will involve choosing how we will divide code into packages and modules. 
+In its current state, application resembles an MVC based architecture. 
+We would like to make a clear decision, what application structure should be going forward.
+
+## Decision
+
+We will divide application into layers represented by packages.
```

Comment:
I think it would be worth mentioning that the app is divided by functionalities first, and then describe the class/package naming conventions, with the responsibilities of each block